scripts/add-new-version.js

```diff
@@ -42,29 +42,29 @@ function addNewSchemaVersion(newVersion, newVersionDir, latestVersion) {
   const obj = require(objFile);
 
   // Adapt all the MUST supported schema formats
-  let mustSupportedSchemaFormats = [] = obj?.else?.properties?.schemaFormat?.anyOf[1]?.enum;
+  let mustSupportedSchemaFormats = obj?.else?.properties?.schemaFormat?.anyOf[1]?.enum || [];
```

Comment:
Ugly, but i can't come with another solution compatible with this version of ES.

You will need to apply the same approach for the rest.

```suggestion
  let mustSupportedSchemaFormats = obj && obj.else && obj.else.properties && obj.else.properties.schemaFormat && obj.else.properties.schemaFormat.anyOf && obj.else.properties.schemaFormat.anyOf[1] && obj.else.properties.schemaFormat.anyOf[1].enum ? obj.else.properties.schemaFormat.anyOf[1].enum : [];
```